src/main/java/net/ossindex/gradle/AuditExtensions.java

```diff
@@ -6,6 +6,7 @@
 import java.util.List;
 
 public class AuditExtensions {
+    public String junitReport = null;

```

Comment:
Minor nit, for fields the "null" is optional. Technically the same may be true for the "failOnError" but I can never remember what the default is ;)

```
public String junitReport;
```

Not really important, more just an FYI.